src/modules/mc_pos_control/PositionControl/PositionControl.hpp

```diff
@@ -181,42 +209,40 @@ class PositionControl : public ModuleParams
 	void _velocityController(const float &dt); /** applies the PID-velocity-controller */
 	void _setCtrlFlag(bool value); /**< set control-loop flags (only required for logging) */
 
-	matrix::Vector3f _pos{}; /**< MC position */
-	matrix::Vector3f _vel{}; /**< MC velocity */
-	matrix::Vector3f _vel_dot{}; /**< MC velocity derivative */
-	matrix::Vector3f _acc{}; /**< MC acceleration */
-	float _yaw{0.0f}; /**< MC yaw */
-	matrix::Vector3f _pos_sp{}; /**< desired position */
-	matrix::Vector3f _vel_sp{}; /**< desired velocity */
-	matrix::Vector3f _acc_sp{}; /**< desired acceleration: not supported yet */
-	matrix::Vector3f _thr_sp{}; /**< desired thrust */
+	// Gains
+	matrix::Vector3f _gain_pos_p; ///< Position control proportional gain
+	matrix::Vector3f _gain_vel_p; ///< Velocity control proportional gain
+	matrix::Vector3f _gain_vel_i; ///< Velocity control integral gain
+	matrix::Vector3f _gain_vel_d; ///< Velocity control derivative gain
+
+	// Limits
+	float _lim_vel_horizontal{0}; ///< Horizontal velocity limit with feed forward and position control
+	float _lim_vel_up{0}; ///< Upwards velocity limit with feed forward and position control
+	float _lim_vel_down{0}; ///< Downwards velocity limit with feed forward and position control
+	float _lim_thr_min{0}; ///< Minimum collective thrust allowed as output [-1,0] e.g. -0.9
+	float _lim_thr_max{0}; ///< Maximum collective thrust allowed as output [-1,0] e.g. -0.1
+	float _lim_tilt{0}; ///< Maximum tilt from level the output attitude is allowed to have
+
+	float _hover_thrust{0}; ///< Thrust [0,1] with which the vehicle hovers not aacelerating down or up with level orientation
+
+	// States
+	matrix::Vector3f _pos; /**< position */
+	matrix::Vector3f _vel; /**< velocity */
+	matrix::Vector3f _vel_dot; /**< velocity derivative (replacement for acceleration estimate) */
+	matrix::Vector3f _thr_int; /**< integral term of the velocity controller */
+	float _yaw = 0.0f; /**< yaw */
```

Comment:
```suggestion
	float _yaw{}; /**< yaw */
```

Reply:
Also, is the comment really useful?

Reply:
I also changed the comment to be useful.